### PR TITLE
[Kotlin Migration] CardBrowserSearchView

### PR DESCRIPTION
**Pull request by puranjayK**
[Kotlin Migration] CardBrowserSearchView
package com.ichi2.ui

### CODE REVIEW

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
CI failure - could you rebase the fix in
This should have been handled in the pre-commit hook

```
* What went wrong:
Execution failed for task ':AnkiDroid:ktlintMainSourceSetCheck'.
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/java/com/ichi2/ui/CardBrowserSearchView.kt:1:1 File must end with a newline (\n)
> A failure occurred while executing org.jlleitschuh.gradle.ktlint.worker.ConsoleReportWorkAction
   > KtLint found code style violations. Please see the following reports:
     - /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/build/reports/ktlint/ktlintMainSourceSetCheck/ktlintMainSourceSetCheck.txt
```

**Review by david-allison-1 — APPROVED** [review]
LGTM

File: AnkiDroid/src/main/java/com/ichi2/ui/CardBrowserSearchView.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+package com.ichi2.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.appcompat.widget.SearchView
+
+class CardBrowserSearchView : SearchView {
+    constructor(context: Context) : super(context) {}
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {}
+    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {}
```
Comment (Akshay0701) [inline AnkiDroid/src/main/java/com/ichi2/ui/CardBrowserSearchView.kt:26]: add kotlinCleanup: `make attrs nonnull`

**Review by Akshay0701 — CHANGES_REQUESTED** [review]
Hey @puranjayK, Thank you for the contributions...

attribute `attrs` from constructor was nonnull in java file, add a kotlinCleanup

File: AnkiDroid/src/main/java/com/ichi2/ui/CardBrowserSearchView.kt (hunk 1 above)
Comment (puranjayK) [reply AnkiDroid/src/main/java/com/ichi2/ui/CardBrowserSearchView.kt]: Done. PTAL

**Review by sud007 — APPROVED** [review]
Changes look fine

**Review by mikehardy — APPROVED** [review]
nice, thank you!

**Comment by mikehardy** [comment]
Hi there! **JANUARY** OpenCollective Notice

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for [January 2022 submissions](https://github.com/ankidroid/Anki-Android/pulls?q=is%3Apr+merged%3A2022-01-01..2022-01-31) (I'll follow this with the same process for February in a week)

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

(I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged for in the month of January)

Thanks!

**Pull request merged**